packages/utils/util.ts

```diff
@@ -257,3 +257,14 @@ export const refAttacher = <T extends HTMLElement | ComponentPublicInstance>(
     ref.value = val
   }
 }
+
+export function createLink(link: string) {
```

Comment:
Suggestion: rename to `openLink`

```suggestion
export function openLink(link: string) {
```